target/linux/layerscape/patches-4.4/0001-arch-Introduce-post-init-read-only-memory.patch

```diff
@@ -0,0 +1,114 @@
+From 7ece66e296972744ec471d521b8ca07990e2494f Mon Sep 17 00:00:00 2001
```

Comment:
This commit and probably many more are already in kernel 4.9, could you please update this target to kernel 4.9.
The next LEDE release will be based on kernel 4.9 anyway so you have to update this target when the layerscape target should be included in the next release.